pkg/apis/core/types.go

```diff
@@ -2171,6 +2177,85 @@ const (
 	VmPowerSpecSuspended VmPowerSpec = "suspended"
 )
 
+// Enables fine grained control of the block device mapping for an instance
```

Comment:
Not sure if this cannot leverage the current pv model in kubernetes and let the runtime ( when create and start the VM ) to figure out the details of the block device ?

Reply:
In the beginning, we decided to map some of the OpenStack Instance Types to Kubernetes Types, and for those fields that cannot be mapped, we added as a new type. In some scenarios, OpenStack Instance can be created by using BlockDeviceMapingV2, but currently, we haven't reached that level.